### PR TITLE
[styles] Add area style for school, college, kindergarten and education landuse 

### PR DESCRIPTION
**Pull request by dvdmrtnz**
I added an area rendering for `amenity=school`. This will fix https://github.com/organicmaps/organicmaps/issues/1221. I used the style from `amenity=university`. This is how it will look like:

### CODE REVIEW

**Comment by biodranik** [comment]
Isn't it also needed for a kindergarten?

File: data/styles/clear/include/Basemap.mapcss [hunk 1]
```diff
@@ -445,6 +446,7 @@ area|z14-[area:highway=living_street],
 area|z14-[amenity=grave_yard],
 area|z14-[landuse=cemetery],
 area|z14-[amenity=university],
+area|z14-[amenity=school],
```
Comment (pastk) [inline data/styles/clear/include/Basemap.mapcss:449]: School grounds are usually much smaller compared to universities, so makes sense to render them at higher zoom levels.
E.g. in Icons.mapcss schools icons are rendered starting from z17-. I'd keep the area fill and icons either in sync or add the fill one zoom level earlier maybe.

**Comment by dvdmrtnz** [comment]
Thanks for your feedback! I will add `amenity=kindergarten`, `amenity=college` and `landuse=education` and will set the zoom levels to be the same as the icons.

**Comment by dvdmrtnz** [comment]
There is a problem with these changes. Adding an area rendering for `amenity=school` is hiding the area rendering for `leisure=pitch`. 

I checked `Basemap.mapcss` and the z-index of `leisure=pitch` is 5 while the z-index of `amenity=school` is 1, so the pitch should be rendering on top of the school area, right? There must be something that I am missing. 

How can I make the pitch render on top of the school area?

See example (https://www.openstreetmap.org/way/32672009)

Before:

Screenshot_20220404-215255

After:

Screenshot_20220404-210749

**Comment by pastk** [comment]
This is because school's z-index is overriden in Icons.mapcss to 2500..
We had the same issue with hospital grounds, there the solution was to remove z-index from Icons, see https://github.com/organicmaps/organicmaps/pull/2166/files#diff-1f979d5595296f97800fd57537b1becb9d74af23f71f5aa3cc5e3c2989fc0ba7

(However now I'm not 100% sure using `-x-me-text-priority:` hack makes sense there, as it looks like captions are using parent icon's priority anyway (in runtime displacement code at least), but should be OK to go this way for now).

**Comment by dvdmrtnz** [comment]
I think it should be good now. This is how it will look:



I also added `landuse=education` with same style.

File: data/styles/clear/include/Basemap.mapcss [hunk 2]
```diff
@@ -56,7 +59,8 @@ area[landuse=construction],
 area[landuse=churchyard],
 area[landuse=landfill],
 area[landuse=railway],
-area[landuse=quarry]
+area[landuse=quarry],
+area[landuse=education],
```
Comment (pastk) [inline data/styles/clear/include/Basemap.mapcss:63]: More specific area like school, university, should have a priority over more generic "education".
I suggest let's move education to z-index:1 and move university, college, etc. to 2.

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -9743,6 +9743,11 @@
     uk = Будівництво
     zh-Hans = 建设区域
 
+  [type.landuse.education]
+    en = Educational Land
+    es = Terreno educativo
+    fr = Zone éducative
```
Comment (pastk) [inline data/strings/types_strings.txt:9749]: en = Educational Facilities
be = Адукацыйныя ўстановы
? es = Instalaciones educacionales
ru = Образовательные учреждения
uk = Освітній заклади

**Review by pastk — CHANGES_REQUESTED** [review]
A little more changes, please!

File: data/styles/clear/include/Basemap.mapcss [hunk 4]
```diff
@@ -479,7 +487,11 @@ area|z14-[amenity=doctors]
 
 /* 7.3 University & Sport */
 
-area|z14-[amenity=university]
+area|z14-[amenity=university],
+area|z17-[amenity=college],
```
Comment (pastk) [inline data/styles/clear/include/Basemap.mapcss:491]: college is z16- now

**Comment by vng** [comment]
Merge conflict. Please, rebase and make 2 commits: first with style changes, second - regenerated styles.

**Comment by dvdmrtnz** [comment]
Should be good now

**Review by vng — APPROVED** [review]

**Review by pastk — APPROVED** [review]
LGTM. Thanks a lot!!

**Pull request merged**